### DISCUSSION TITLE
FIC calculation

### DISCUSSION THREAD

**Discussion by fezegolshani** [firemodels/fds#12931] [General]
Hello, 
 My thesis focuses on fire evacuation, and I utilize PyroSim to capture the dynamics of fire. I have a problem, and this is why I am contacting you. I need to capture the FIC values over simulation time. However, I received zero values for FIC while running my model ( I added a thermocouple device and selected FIC in the drop-down menu). Could you please let me know how I can fix this problem? I can share my model with you if need be.

**Comment by mcgratta** [comment]
Create a very simple test case that demonstrates the problem. This is usually the best way to diagnose the issue.
